# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/performance/PerformanceWorker.cpp
# Commit: cbe7670b4ae5
# Full Hash: cbe7670b4ae5f062939d18ae24af9cc2d7a8da23
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-01-25 16:37:35
# Regressor Bug: 1744025
# File Overlap Count: 3
# Description:
#   Bug 1744025: Use CheckedUnsafePtr<WorkerPrivate> in WorkerGlobalScopeBase. r=dom-worker-reviewers,asuth
#   
#   Depends on D132708
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132709
# ==============================================================================

diff -r dfb7e4b76f92 -r cbe7670b4ae5 dom/performance/PerformanceWorker.cpp
--- a/dom/performance/PerformanceWorker.cpp	Tue Jan 25 08:52:59 2022 +0000
+++ b/dom/performance/PerformanceWorker.cpp	Tue Jan 25 08:53:00 2022 +0000
@@ -5,7 +5,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "PerformanceWorker.h"
-#include "mozilla/dom/WorkerPrivate.h"
+#include "mozilla/dom/WorkerScope.h"
 #include "mozilla/StaticPrefs_dom.h"
 
 namespace mozilla::dom {